tests/unit/effects/core.js

```diff
@@ -132,18 +137,19 @@ asyncTest( "animateClass works with colors", function() {
 	test.toggleClass( "testChangeBackground", {
 		duration: 2000,
 		complete: function() {
-			present( test.css( "backgroundColor" ), [ "#ffffff", "#fff", "rgb(255, 255, 255)" ], "Color is final" );
-			start();
+			assert.present( test.css( "backgroundColor" ), [ "#ffffff", "#fff", "rgb(255, 255, 255)" ], "Color is final" );
+			ready();
 		}
 	} );
 } );
 
-asyncTest( "animateClass calls step option", 1, function() {
+QUnit.test( "animateClass calls step option", 1, function( assert ) {
```

Comment:
Could you convert the expect argument to assert.expect calls as well? (3rd attempt)